zerver/lib/bugdown/__init__.py

```diff
@@ -976,9 +994,16 @@ def run(self, root: Element) -> None:
                       class_attr=class_attr,
                       already_thumbnailed=True)
                 continue
+
             if self.is_image(url):
+                image_source = self.corrected_image_source(url)
+                if image_source is not None:
+                    corrected_url = ResultWithFamily(found_url.family, (image_source, image_source))
```

Comment:
I would spell this with `family=found_url.family`, etc., rather than using positional arguments, since that's what we do elsewhere.